keyboards/hhkb_yang/readme.md

```diff
@@ -0,0 +1,62 @@
+# HHKB Alternate Controller (YANG HHKB BLE Mod)
+
+![YANG HHKB BLE Mod](https://i.imgur.com/aZP1GYc.jpeg)
+
+An alternative controler for the HHKB designed by YANG (yangdigi)
+based on the hasu controller.
+
+Keyboard Maintainer: [Kan-Ru Chen](https://github.com/kanru)  
+Hardware Supported: YANG HHKB BLE Controller  
+Hardware Availability: https://kbdfans.com/products/hhkb-ble-mod-upgrade-module
+
+Make example for this keyboard (after setting up your build environment):
+
+    make hhkb_yang:default
+
+To flash, first boot your keyboard into bootloader (hold ESC and attach usb cable):
+
+    make LUFA_DEVICE=/dev/sda hhkb_yang:default:flash
+
+LUFA_DEVICE should point to the emulated storage device by the lufa-ms
+bootloader. **Caution**: if set to incorrect device it may wipe out
+your actual disk.
```

Comment:
```suggestion
    make LUFA_DEVICE=/dev/sda hhkb_yang:default:flash

LUFA_DEVICE should point to the emulated storage device by the lufa-ms
bootloader. **Caution**: if set to incorrect device it may wipe out
your actual disk.
```

Even though it has a warning text, I'd rather not explicitly mention `/dev/sda`, as most will have that device and in most cases that's not gonna be the one they want to `dd`.